src/main/java/de/bonndan/nivio/output/layout/OrganicLayouter.java

```diff
@@ -0,0 +1,34 @@
+package de.bonndan.nivio.output.layout;
+
+import de.bonndan.nivio.model.Group;
+import de.bonndan.nivio.model.LandscapeImpl;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Applies {@link FastOrganicLayout} to landscape components and writes the rendered data to component labels.
+ *
+ *
+ */
+public class OrganicLayouter implements Layouter<LayoutedComponent> {
+
+    public OrganicLayouter() {
+    }
+
+    @Override
+    public LayoutedComponent layout(LandscapeImpl landscape) {
+
+        Map<String, SubGraph> subgraphs = new LinkedHashMap<>();
```

Comment:
Do we need a LinkedHashMap, or could it be a HashMap? (I don't think that we have memory problems with nivio, just to understand)

Reply:
It is used to ensure that insertion sequence is maintained. Otherwise layout might receive the items in random order, leading to random results.